lms/djangoapps/discussion/rest_api/tests/test_render.py

```diff
@@ -56,33 +56,38 @@ def test_allowed_a_tag(self, protocol):
 
     def test_disallowed_a_tag(self):
```

Comment:
I think this test does not make sense now? Since we allow the a tag

Reply:
The `a` tag was always allowed, but it's just how it's processed that's changed. This test was checking which protocols are disalllowed as @Agrendalath mentioned. I will rename it accordingly.